test/storage_test.rb

```diff
@@ -44,4 +44,24 @@ def test_cannot_write_to_history_storage
     history_storage = TTNT::Storage.new(@repo, sha)
     assert_raises { history_storage.write!(@section, @data) }
   end
+
+  def test_storage_file_resides_with_rakefile
+    @storage.write!(@section, @data)
+    subdir = "#{@repo.workdir}/tmp"
+    Dir.mkdir(subdir)
+    rakefiles = ["#{@repo.workdir}/Rakefile", "#{subdir}/Rakefile"]
+    FileUtils.copy rakefiles[0], rakefiles[1]
+
+    TTNT.instance_variable_set(:@root_dir, nil)
```

Comment:
What do you think about creating a method for this ?

~~~ruby
TTNT.root_dir = nil
~~~